docs/names-and-ids.md

```diff
@@ -0,0 +1,43 @@
+# Names and IDs
+
+version .1
+
+There are a number of names and IDs used in THIEF.  This table describes some of them:
+
+| Identifier | format | Meaning |
+| - | - | - |
+| servicePool | string | Name of pool that a service app belongs to. |
+| requestedServicePool | string | Name of pool that a device app would like to pair with. |
+| runId | guid | Guid representing a running executable. |
+| serviceAppRunId | guid | Guid representing a running _service_ executable. |
+| sessionId | guid | Guid representing the pairing between a device app and a service app. |
```

Comment:
How about `pairingId` then?

Reply:
it was pairingId, but that wasn't quite right.  See my comment on collapsing runId and sessionId into a single value.  I think that will be less confusing.

Reply:
Well, presumably a device starts up, and gets a pairing. If it keeps that pairing for the duration of the lifetime, then the device run Id could serve the same purpose. However, if it could lose a service and pick up another then you'd want another value: pairing Id.